symbols/overlay29.yml

```diff
@@ -876,6 +886,17 @@ overlay29:
         Advances one frame. Does not return until the next frame starts.
         
         r0: ? - Unused by the function
+    - name: DisplayDamageNumbers
```

Comment:
I believe this function has a niche use for the Stockpile effect in 0x22E46D4 (NA). I doubt it's worth generalizing the function name, but it's probably worth noting.

Reply:
How is it used for Stockpile? If it's just the same damage number effect, I feel that it's covered by the general case and maybe not worth noting. It might make more sense to discuss this in a comment for the `NUMBER_COLOR_LIGHT_BLUE` value you suggested in the headers.

Reply:
Oh right, I didn't remember that. I'll change it.

Reply:
Alright, new description looks fine.